code/modules/mob/living/simple_animal/bot/mulebot.dm

```diff
@@ -829,6 +828,16 @@
 	if(.)
 		visible_message("<span class='notice'>[src]'s safeties are locked on.</span>")
 
+/mob/living/simple_animal/bot/mulebot/bot_step(dest)
```

Comment:
this only occurs during astar movements. power should still be drained while the bot is under player control.